roles/openshift_provisioning_aws_facts/README.md

```diff
@@ -0,0 +1,43 @@
+OpenShift Provisioning AWS Facts
+=========
+
+Provides common facts and defaults for other AWS provisioning roles.
+
+Requirements
+------------
+
+None
+
+Role Variables
+--------------
+
+| Name                                   | Default value                                       |                     |
+|----------------------------------------|-----------------------------------------------------|---------------------|
+| openshift_cluster_id                   | default                                             | Cluster ID (combination of openshift_cluster_id and openshift_env_id are expected to be unique for all OpenShift clusters) |
```

Comment:
@dgoodwin I think we would require that a user explicitly set the variables, I think we might want to default them in the current byo/config.yml playbook for backward compatibility, but it would need to be explicitly set for new playbooks that would leverage the provisioning.